smart-contract/src/lottery.rs

```diff
@@ -199,6 +202,7 @@ impl Lottery {
                 // update play event state
                 is_jackpot = true;
                 prize = prize_value;
+                jackpot_amount = prize_value;
```

Comment:
Since it's `/// Current jackpot amount after play` the `jackpot_amount` should be `0` in such a case